protos/shared/shared.proto

```diff
@@ -29,9 +33,10 @@ message RuntimeInfoReport {
   bytes git_hash = 7; //the git hash of the build related the git repository's branch
   bytes version = 8;
   bytes db_status_extra = 9; //RESERVED: JSON format structure to store db status information
-  bytes queue_status_extra = 10; //RESERVED: JSON format structure to store the queue information
-  bytes chain_status_extra = 11; //RESERVED: JSON format structre to store the block chain related information
-  bytes cache_status_extra = 12; //RESERVED: JSON format structre to store the cache related information
-  bytes extra = 13; //RESERVED: JSON format structre to store other information
-  bytes signature = 14;
+  bytes rd_status_extra = 10; //RESERVED: JSON format structure to store redis status information
+  bytes queue_status_extra = 11; //RESERVED: JSON format structure to store the queue information
+  bytes chain_status_extra = 12; //RESERVED: JSON format structre to store the block chain related information
+  bytes cache_status_extra = 13; //RESERVED: JSON format structre to store the cache related information
+  bytes extra = 14; //RESERVED: JSON format structre to store other information
+  bytes signature = 15;
```

Comment:
Here the signature need to move out like this,
message RuntimeInfoRequest {
  ...
  ...
}
message SignedRuntimeInfoRequest {
  RuntimeInfoRequest req = 1;
  bytes signature = 2;
}
Signature is signed for the "RuntimeInfoRequest req" message.

Reply:
ok